docs/cli.md

```diff
@@ -688,10 +694,23 @@ gcli group grant --group groupA --role admin
 ```
 
 #### Remove a role from a group
+
 ```bash
 gcli group revoke --group groupA --role admin
 ```
 
+### Grant a privilege
+
+```bash
+gcli catalog grant --name catalog_postgres --role admin --privilege create_table modify_table
```

Comment:
I think the "grant privilege" action should be executed on the role entity, instead of at a catalog/schema/table entity, because we grant privilege of a securable object to role first, and then grant the role to a specific user or group.

IMO, the command should be like:
"gcli role grant --role admin --name catalog_postgres --privilege create_table modify_table"
"gcli role grant --role admin --name catalog_postgres.default --privilege create_table modify_table"
"gcli role grant --role admin --name catalog_postgres.default.hr --privilege select_table"
"gcli role revoke --role admin --name catalog_postgres --privilege create_table modify_table"

@jerqi can confirm this behavior.

Reply:
This works the same way and is in the same format as the owner command as it deals with an entity and granting or revoking a privilege on it.

Reply:
> I think the "grant privilege" action should be executed on the role entity, instead of at a catalog/schema/table entity, because we grant privilege of a securable object to role first, and then grant the role to a specific user or group.
> 
> IMO, the command should be like: "gcli role grant --role admin --name catalog_postgres --privilege create_table modify_table" "gcli role grant --role admin --name catalog_postgres.default --privilege create_table modify_table" "gcli role grant --role admin --name catalog_postgres.default.hr --privilege select_table" "gcli role revoke --role admin --name catalog_postgres --privilege create_table modify_table"
> 
> @jerqi can confirm this behavior.

Maybe we should add the type of the object, too.

Reply:
The type of entity is specified as the first command argument.

Reply:
Also, note that previous feedback dropped the option of having a flag to specify an object type.

Reply:
Sorry I accidentally closed this

Reply:
To do that the Java API will need to change first.

Reply:
I checked the Java API, I think the Java API is okay (https://github.com/apache/gravitino/blob/main/clients/client-java/src/main/java/org/apache/gravitino/client/GravitinoClient.java#L423). The only thing is, you need to construct a MetadataObject which need to know the entity type. But even without the type, you can get that easily by invoke the Java API to get the catalog object, then knowing it is a catalog, schema, table, fileset or topic, is it?

@jerqi @jerryshao your comments are welcomed as well.